tests/conftest.py

```diff
@@ -303,26 +303,41 @@ def _slurm_not_installed():
 def _subprocess_setup(request, cfg=None, uwsgi=False):
     """setup the supervisor"""
     import os
+    import sirepo.job
+    import sirepo.pkcli.service
     from pykern.pkcollections import PKDict
 
+    ports = PKDict()
```

Comment:
It could be argued that ports should be an object so we don't have the unrelated values. I'm not sure the complexity is worth it...